docs/en/installation.md

```diff
@@ -38,19 +39,43 @@ source activate xrmocap
 conda install -y ffmpeg
 
 # install pytorch
-conda install -y pytorch==1.8.1 torchvision==0.9.1 cudatoolkit=10.1 -c pytorch
+conda install pytorch==1.12.0 torchvision==0.13.0 cudatoolkit=11.3 -c pytorch
 
```

Comment:
Shall we keep the old version of the `installation.md`?

Reply:
Fixed in [2e69819](https://github.com/openxrlab/xrmocap/pull/71/commits/2e6981933fd4c6ca0680021655faa4820900cc15).